### DISCUSSION TITLE
Sunbird video player for DIKSHA - Version used and branch

### DISCUSSION THREAD

**Discussion by pushpamtrigyn** [Sunbird-Knowlg/Community#162] [Documentation Issues]
Hi,

Currently DIKSHA application is using the following Sunbird video player version :-

"@project-sunbird/sunbird-video-player-v9": "5.1.3",

I was able to trace the repository to https://github.com/Sunbird-Knowlg/sunbird-video-player. However, for the v9 version and 5.1.3 npm package version, I am not sure from which branch or tag this was built from.

We actually require to modified some code, so require to fork the corresponding repo and make changes to it. Please assist for the same.

CC - @vinukumar-vs @pallakartheekreddy

**Comment by pallakartheekreddy** [comment]
@pushpamtrigyn Please use https://github.com/Sunbird-Knowlg/sunbird-video-player/commits/release-5.2.1

**Reply by pallakartheekreddy** [reply]
@pushpamtrigyn Hope your query is now resolved. Please mark the thread as answered.

**Comment by pushpamtrigyn** [comment]
Thanks @pallakartheekreddy
